src/tutorials/0007-verifying-storage-on-filecoin/02.md

```diff
@@ -0,0 +1,25 @@
+---
+    title: "Preparing and transferring data"
+    type: "multiple-choice" # can be "text", "code", "file-upload" or "multiple-choice"
+---
+
+## Preparing data for storage
+
+Before a system file (e.g. `puppy.gif`) can be stored on the Filecoin network, it must first be transformed into a Filecoin Piece.
+
+In the first stage of this transformation, the system file is chunked up with UnixFS to create an IPLD DAG (Directed Acyclic Graph). You can learn more about DAGs (a form of merkle tree) in our [Decentralized Data Structures](https://proto.school/#/data-structures) tutorial. This IPLD DAG has a _payload CID_, identical to an IPFS CID, which represents the root of the DAG.
+
+The IPLD DAG is then serialized to a [CAR file](https://github.com/ipld/specs/blob/master/block-layer/content-addressable-archives.md#summary) and bitpadded to make a Filecoin Piece. This piece has a unique _piece CID_, also known as a CommP (Piece Commitment).
+
+Since payload CIDs and piece CIDs are cryptographic hashes of the data itself, they're unique, with identical CIDs guaranteeing identical content. Identical IPLD DAGs will produce identical payload CIDs and identical pieces will produce identical piece CIDs, no matter who is going to store or retrieve them.
+
+## Negotiating a storage deal and transferring data
```

Comment:
Maybe mention the current sizes of sectors and that for now you can only store data up to a single sector size per deal? (at least I think that is the current state)

Reply:
@mishmosh Would you like me to add this or leave as is? We could perhaps link to the right section of the docs instead of adding much here.

Reply:
Maybe add a small note about it, but link to the docs for more info: https://docs.filecoin.io/how-to/store-prepare-data/#for-files-bigger-than-a-sector

Reply:
Thanks! Have added this: `Sectors come in a variety of sizes, and a client can store data up to the largest sector size per deal. (Learn more about [sector sizes and storing large files](https://docs.filecoin.io/how-to/store-prepare-data/#for-files-bigger-than-a-sector).)`